packages/pkp-ethers/src/lib/pkp-ethers.ts

```diff
@@ -88,16 +87,12 @@ export class PKPEthersWallet
       prop.rpc || RPC_URL_BY_NETWORK[prop.litNodeClient.config.litNetwork];
 
     if (!rpcUrl) {
-      throw new Error('No RPC URL provided');
```

Comment:
Removed some duplicate code here that probably ended up here due to merge conflict shenanigans